spanner/api/Spanner.Samples/ListInstanceConfigs.cs

```diff
@@ -19,27 +19,35 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Threading.Tasks;
 
 public class ListInstanceConfigsSample
 {
-    public IEnumerable<InstanceConfig> ListInstanceConfigs(string projectId)
+    public async Task<IEnumerable<InstanceConfig>> ListInstanceConfigsAsync(string projectId)
     {
-        var instanceAdminClient = InstanceAdminClient.Create();
+        var instanceAdminClient = await InstanceAdminClient.CreateAsync();
         var projectName = ProjectName.FromProject(projectId);
-        var instanceConfigs = instanceAdminClient.ListInstanceConfigs(projectName);
-
+        var instanceConfigs = instanceAdminClient.ListInstanceConfigsAsync(projectName);
+        
         // We print the first 5 elements for demonstration purposes.
         // You can print all configs in the sequence by removing the call to Take(5).
         // The sequence will lazily fetch elements in pages as needed.
-        foreach (var instanceConfig in instanceConfigs.Take(5))
+        await foreach(var instanceConfig in instanceConfigs.Take(5))
```

Comment:
But otherwise I won 't have an extension method .Take for AsyncEnumerable

Reply:
Ah yes, that's right, sorry for missing that.

Hmmm, then I think we can keep that part of the method sync for now. It will match all out other listing samples and we don't need to add the extra dependency. Once we have sorted out our quota issues in a different way than just printing 5 elements, we can revisit all listing operations.